<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>1072</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eik Fest</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
      QDialog#MainWindow {
        background-color: white;  /* ili koju god boju želiš */
      }
    </string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <spacer name="verticalSpacerTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalCenterLayout">
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="minimumSize">
        <size>
         <width>720</width>
         <height>1000</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
      background-color: white;
    </string>
       </property>
       <property name="currentIndex">
        <number>4</number>
       </property>
       <widget class="QWidget" name="WelcomePage">
        <layout class="QVBoxLayout" name="verticalLayout_welcome">
         <item>
          <widget class="QLabel" name="MainLogo">
           <property name="pixmap">
            <pixmap>graphs/logo.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="LoginPage">
        <layout class="QVBoxLayout" name="verticalLayout_login">
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QLabel" name="MainLogo_2">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
           <property name="pixmap">
            <pixmap>graphs/logo.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QGroupBox" name="LoginGroup">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>350</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox {
                border: none;
                margin: 0px;
                padding: 0px;
            }
            QGroupBox::title {
                subcontrol-origin: margin;
                left: 10px;
                padding: 0 0 0 0;
            }
            QLabel {
            font-size: 10pt;
            /* ostala pravila */
            }
            QLineEdit {
            border: 2px solid #E1B10D;
            border-radius: 16px;
            padding: 6px;
            color:black;
            font-size:16px;
            }
            QPushButton{
                background-color: #FFD33E;
                border-radius: 16px;
                border: none;
                font-size: 18px;       /* Smanji ako ti treba manje */
                padding: 10px 0;       /* Smanji padding */
                color: #242424;
                font-family: &quot;Inter&quot;, sans-serif;
                font-weight: 300;   
            }</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="LoginLabel">
              <property name="styleSheet">
               <string notr="true">font-family: Inter;
                        font-weight: 700;
                        font-style: Bold;
                        font-size: 24px;
                        leading-trim: NONE;
                        line-height: 100%;
                        letter-spacing: 0%;
                        text-align: center;
                        text-transform: uppercase;
                        color: #E1B10D;
                        margin-bottom: 20px;
                    </string>
              </property>
              <property name="text">
               <string>PRISTUPI SVOM NALOGU</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="EmailLabel">
              <property name="styleSheet">
               <string notr="true">margin-bottom: 5px;</string>
              </property>
              <property name="text">
               <string>Email adresa</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EmailInput">
              <property name="styleSheet">
               <string notr="true">margin-bottom: 15px;</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>redwings@vibe2025.rs</string>
                <!-- <string>prodajnomesto1@vibe2025.rs</string> -->
              </property>
              <property name="cursorPosition">
               <number>20</number>
              </property>
              <property name="placeholderText">
               <string>Unesi svoji email adresu ovde</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="PswLabel">
              <property name="styleSheet">
               <string notr="true">margin-bottom: 5px;</string>
              </property>
              <property name="text">
               <string>Šifra</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PswInput">
              <property name="styleSheet">
               <string notr="true">margin-bottom: 20px;</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>12345678</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="LoginButton">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Uloguj se &gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="CardPage">
        <layout class="QVBoxLayout" name="verticalLayout_login" stretch="5,1,1,5,1">
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QLabel" name="MainLogo_3">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="pixmap">
            <pixmap>graphs/logo.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ReadCardLabel">
           <property name="styleSheet">
            <string notr="true">color: #E1B10D;
                    font-family: Inter;
                    font-weight: 700;
                    font-style: Bold;
                    font-size: 28px;
                    leading-trim: NONE;
                    line-height: 100%;
                    letter-spacing: 0%;
                    text-align: center;
                    text-transform: uppercase;



                    </string>
           </property>
           <property name="text">
            <string>Očitaj karticu</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ReadCardPara">
           <property name="styleSheet">
            <string notr="true">font-family: Inter;
                    font-weight: 400;
                    font-style: Regular;
                    font-size: 18px;
                    leading-trim: NONE;
                    line-height: 24px;
                    letter-spacing: 0%;
                    text-align: center;
                    color:black;
                    </string>
           </property>
           <property name="text">
            <string>Prislonite karticu uređaju kako bi se očitala i time prebacila u aktivno stanje.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ReadCardGraph">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>graphs/scancard.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="CardHelpFooter">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>graphs/pomoc.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="CatalogPage">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>900</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>900</width>
          <height>1100</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_login" stretch="1,1,10,2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QLabel" name="catalogLabel">
           <property name="styleSheet">
            <string notr="true">    color: #E1B10D;
                                        font-family: Inter;
                                        font-weight: 700;
                                        font-style: Bold;
                                        font-size: 28px;
                                        text-align: center;
                                        text-transform: uppercase;  
                                    </string>
           </property>
           <property name="text">
            <string>FESTIVALSKI MENI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="categoryScrollArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>QScrollBar:horizontal {
                                        height: 6px;
                                        border-radius: 3px;
                                    }
                                    QScrollBar::handle:horizontal {
                                        border-radius: 3px;
                                    }</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="styleSheet">
             <string notr="true">
                        background-color: white;
                    </string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="mainHorizontalLayout" stretch="6,4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="productScrollArea">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
                        background-color: white;
                    </string>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustIgnored</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>429</width>
                <height>727</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="BasketScrollArea">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>4</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
                        background-color: transparent;
                    </string>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustIgnored</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>285</width>
                <height>727</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="bottomLayout" stretch="3,7">
           <property name="spacing">
            <number>16</number>
           </property>
           <property name="leftMargin">
            <number>16</number>
           </property>
           <property name="topMargin">
            <number>16</number>
           </property>
           <property name="rightMargin">
            <number>16</number>
           </property>
           <property name="bottomMargin">
            <number>16</number>
           </property>
           <item>
            <widget class="QPushButton" name="cancelOrderButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>64</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
                        QPushButton {
                            height: 64px;
                            background: #E11B1B;
                            font-family: Inter;
                            font-weight: 600;
                            font-size: 20px;
                            line-height: 24px;
                            color: white;
                            border-radius: 16px;
                            border: none;
                        }
                        QPushButton:pressed {
                            background: #A11212;
                        }</string>
             </property>
             <property name="text">
              <string>Otkaži</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="finishOrderButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>64</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
                                                    height: 64px;
                                                    background: #FFD33E;
                                                    font-family: Inter;
                                                    font-weight: 600;
                                                    font-size: 20px;
                                                    line-height: 24px;
                                                    color: #474747;
                                                    border-radius: 16px;
                                                    border: none;
                                                }
                                                QPushButton:pressed {
                                                    background: #E6BE35;
                                                }</string>
             </property>
             <property name="text">
              <string>Završi račun</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>

      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
